astropy/coordinates/transformations.py

```diff
@@ -744,8 +748,190 @@ def __call__(self, fromcoord, toframe):
                 'should have been of type {1}'.format(res, self.tosys))
         return res
 
+class BaseAffineTransform(CoordinateTransform):
+    """
+    Base class for common functionality between the ``AffineTransform``-type
+    subclasses.
```

Comment:
Maybe add some clarification here for why there needs to be a base?  (As opposed to just having `AffineTransform`)  I can imagine someone wanting to subclass being confused which to subclass from...